flux.yaml

```diff
@@ -111,7 +111,7 @@ items:
         - args:
           - --ssh-keygen-dir=/var/fluxd/keygen
           - --git-url=https://github.com/weaveworks/wks-quickstart-firekube.git
-          - --git-branch=master
+          - --git-branch=fix-0-8-4
```

Comment:
Will this be a lasting change?

Reply:
it caused by 1256767.

Reply:
Still not sure about this change though......?